src/domWrapper.ts

```diff
@@ -67,6 +56,47 @@ export class DOMWrapper<ElementType extends Element>
     )
   }
 
+  findComponent<T extends ComponentPublicInstance>(
+    selector: FindComponentSelector | (new () => T)
+  ): VueWrapper<T> {
+    const parentComponent: ComponentInternalInstance = (this.element as any)
+      .__vueParentComponent
```

Comment:
maybe it would be better to define `wrapperElement` as `private readonly wrapperElement: ElementType & { __vueParentComponent: ComponentInternalInstance }` or something like that. THis would avoid casting `as any` here.